MembraneVideoroomDemo/Controllers/RoomController.swift

```diff
@@ -393,15 +457,54 @@ extension RoomController: MembraneRTCDelegate {
         }
     }
 
-    func onTrackUpdated(ctx _: TrackContext) {}
+    func onTrackUpdated(ctx: TrackContext) {
+        let isActive =
+            (ctx.metadata["active"] != nil)
+            ? ctx.metadata["active"] as! Bool : ctx.metadata["type"] as! String == "camera" ? false : true
+
+        if ctx.metadata["type"] as! String == "camera" {
```

Comment:
I'm not a fan of ! and `as!` operators. We know that it's most likely a string but if it's null or something for some reason (we don't have control over videoroom web frontend) it's going to crash the app.

Reply:
Ack.